app/assets/stylesheets/components/_moj.cookie_banner.scss

```diff
@@ -0,0 +1,62 @@
+.js-enabled #cookie-message {
+  display: block; /* shown with JS, always on for non-JS */
+}
+.js-enabled #global-cookie-message {
+  display: none !important;
+}
+
+#cookie-message {
+  @extend %site-width-container;
+  @include core-19;
+
+  display: none;
+  padding: $gutter 0 (2 * $gutter);
+  color: $green;
+
+  .js-enabled & {
+    display: block;
+  }
+
+  form a.mtp-cancel-button {
+    color: $green;
+
+    &:visited {
+      color: darken($green, 10%);
+    }
+  }
+
+  .button {
+    border: 2px solid $green;
+    color: #fff;
+    background-color: $green;
+    box-shadow: none;
+
+    &:focus {
+      outline: none;
+      color: $text-colour;
+      background: $focus-colour;
+      border-color: $focus-colour;
+    }
+  }
+
+  @include media(mobile) {
+    .button {
+      width: 40%;
+    }
+
+    .mtp-cancel-button {
+      display: block;
+      margin-top: 1em;
+    }
+  }
+}
+
+#id_accept_cookies-wrapper {
+  margin-top: $gutter-half;
+}
+#accept-cookies {
+  margin-right: 10;
+}
+#reject-cookies {
+  margin-right: 10;
+}
```

Comment:
nit: missing [EOF newline character](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

Reply:
done